content/cucumber/configuration.md

```diff
@@ -22,17 +84,15 @@ want to execute with this profile.
 The example above generates two profiles:
 
 1. `html_report`, with a list of command-line options that specify new output formats, and
-2. `bvt`, which executes all Features and Scenarios [tagged](/cucumber/api/#tags) with `@bvt`.
+2. `bvt`, which executes all Features and Scenarios [tagged](/cucumber/api#tags) with `@bvt`.
 
-{{% text "ruby" %}}
 Cucumber-Rails creates a `cucumber.yml` file in the project config directory containing a number of predefined profiles,
 one of which is the default profile. When Cucumber is run from the command line, it is usually necessary to provide both
 the directory name containing the root directory of the tree containing feature files and the directory name containing
 references to the necessary library files. In a typical project, `cucumber --require features features/some/path` will suffice.
 Repetitious usages can be added to user-defined profiles contained in the project's `cucumber.yml` file.
-{{% /text %}}
 
-## Executing Profiles
+<b>Profile can be executed</b>
```

Comment:
You might not be aware, but we agreed earlier to not have headers that look like headers but aren't technically headers. We also do not allow headers inside language blocks (as you probably found out). The fact that we have headers here that are / seem language specific is a hint that the page should be split into language specific pages.

Reply:
That is the problem. This stuff needs a header. But it is very specific to ruby and has no equivalent elsewhere.

Reply:
Which is why I propose to have language specific configuration pages, like here: https://docs.cucumber.io/tools/general/ (one general page, pointing to language specific pages; each language specific page can have it's own headers)